app/views/static/cloud_volume_backup/volume_select.html.haml

```diff
@@ -0,0 +1,23 @@
+%form#form_div{:name => "angularForm",
+               "miq-form"      => true,
+               "model"         => "vm.cloudVolumeBackupModel",
+               "model-copy"    => "vm.modelCopy"}
+  = render :partial => "layouts/flash_msg"
+  %h3
+    = _('Basic Information')
+  .form-horizontal
+    .form-group
+      %label.col-md-2.control-label
+        = _('Volume')
+      .col-md-8
+        = select_tag('volume_id',
+                '',
+                'ng-model'                   => 'vm.cloudVolumeBackupModel.volume',
+                'ng-options'                 => 'volume.name for volume in vm.volume_choices',
+                :id                          => 'volume_id',
+                :required                    => '',
+                'pf-select'                  => true)
+        :javascript
```

Comment:
Can you drop that `:javascript` `miqInitSelectPicker();` please? It's no longer needed when using `pf-select` :)